deploy/updater/updater.py

```
@@ -9,324 +9,257 @@
+    "Дніпропетровська область": 19,
+    "Полтавська область": 20,
+    "Черкаська область": 21,
+    "Кіровоградська область": 22,
+    "Вінницька область": 23,
+    "Хмельницька область": 24,
+    "Чернівецька область": 25,
+    "м. Київ": 26,
+}
+
+
+async def get_cache_data(mc, key_b, default_response={}):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)